src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -285,16 +285,53 @@ size_t DecoderProto::get_input_size() const {
     return m_node_def->input_size();
 }
 
-void DecoderProto::get_input_node(size_t input_port_idx,
-                                  std::string& producer_name,
-                                  size_t& producer_output_port_index) const {
-    // Body graph nodes may have two colons `:`, for example,
-    // producer_name:z:2 means that producer operation name is `producer_name`
-    // and output port is 2
-    std::string producer_port_name = m_node_def->input(static_cast<int>(input_port_idx));
+void parse_producer_name(const std::string& producer_port_name,
```

Comment:
Don't see documentation, check please

Reply:
below you can see a couple of comments what this function is doing. And step-by-step it explains this in detail

Reply:
It doesn't describe a function definition, overall functionality and expected parameters.